engine/README.md

```diff
@@ -198,3 +198,66 @@ mono bee.exe mac
 ```
 
 
+## build android sdk on mac
```

Comment:
try to make the section names and structures similar to those of other platforms

try also fix the typos: 
flutter-1.18-candidate.5 =>flutter-1.17-candidate.5